### ISSUE TITLE
Traversed fields not showing up in List Reports

### ISSUE THREAD

**Issue by oaaoaa** [ra-systems/django-slick-reporting#76]
```
class TestList(ListReportView):
    report_title = "Test List"
    report_model = MyModel # has a foreign key fk_field which is a model with a field 'a'
    date_field = "dt"

    columns = [("dt", {"verbose_name": "Date"}),
               ("fk_field__a", {"verbose_name": "Alien Field"}),
               ("field1", {"verbose_name": "MyModel Field"}),]

```

The column under Alien Field renders as blank because list reports are not able to read and include values of traversed fields. I could come up with a work around by changing the second column to "fk_field" and then traversing the foreign key object myself in format_row to read the appropriate field. But that is a bit inconvenient. Therefore, traversed fields should be read and rendered in List Reports just like any other model fields.

The columns specified in List Reports are also checked at the time of initialising the class to ensure that all belong to the model, attribute of generator etc. There is no need for doing any column checks in List Reports as we are not doing any aggregation/grouping on any of the fields. Let the developer be given the freedom to specify any thing as a column (even non fields) and make it the responsibility of the developer to populate it in format_row or through a callback.

**Comment by RamezIssac** [comment]
Hello 
Yes this issue with traversing fields in ListReportView is noted.
Thank you for the report, will address it asap.

**Comment by RamezIssac** [comment]
Hello @oaaoaa 
Published a new release 1.1.0
This issue should be fixed
